workshops/templates/workshops/workshop_issues.html

```diff
@@ -36,7 +36,7 @@
         {% if event.mailto_ %}
           <a href="mailto:{{event.mailto_}}?subject={% filter urlencode %}Missing location for workshop {{event.get_ident}}{% endfilter %}&body={% filter urlencode %}Hi,
 
-Can you please send us the location where the {{ event.get_ident }} workshop took place?
+Can you please send us the location where the {{ event.get_ident }} workshop took place? We need a country, address, venue, and latitude/longitude.
```

Comment:
I'd want to understand why we are collecting this for past workshops.  Longitudinally I think we want to track workshops geographically i.e. how many in Florida? how many in south east US? etc.  But the question is how specific do we need to get? @jduckles ?
 
Obviously for upcoming workshops we want to know the exact location down to the street address so people who want to attend can find it.

For past workshops, can we just have the host - i.e., "University of Miami" - or does it matter in the long term exactly what building on Miami's campus it took place?  

We have no workshops currently missing location so I don't have a real test case (https://amy.software-carpentry.org/workshops/reports/workshop_issues).